test/mocks/upstream/cluster_info.cc

```diff
@@ -61,7 +61,7 @@ MockClusterInfo::MockClusterInfo()
           runtime_, "fake_key", 1, 1024, 1024, 1, std::numeric_limits<uint64_t>::max(),
           circuit_breakers_stats_, absl::nullopt, absl::nullopt)),
       stats_scope_(stats_store_.createScope("test_scope")) {
-  ON_CALL(*this, connectTimeout()).WillByDefault(Return(std::chrono::milliseconds(1)));
+  ON_CALL(*this, connectTimeout()).WillByDefault(Return(std::chrono::milliseconds(5001)));
```

Comment:
default 1ms connect timeout is definitely unreasonable, but does it somehow cause the test fail?  Anyway we probably need to do sanity check on the configured value, could you file an issue about the problem you hit?

Reply:
I think this value will be converted to QUIC handshake timeout.
And I ran into the QUIC bug at https://github.com/google/quiche/blob/main/quic/core/quic_connection.cc#L4634